frontend/best_practises.md

```diff
@@ -80,8 +82,9 @@ export const MissionTitleComponent = ({ missionName } : MissionTitleComponentPro
 ```
 
 Bad:
+
 ```
-export const RobotComponent = ({ mission, setColor } : {mission: CondensedMissionDefinition, setColor: (c: string) => void}) => {
```

Comment:
Though I understand that this is needed to be in the same commit as the other frontend changes